components/calendar_components/TaskList.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import Task from "./Task";
+
+export default function TaskList({ tasks, toggleTask }) {
+  return (
+    <div className="task-list">
+      <div className="lefttasks">
+        {tasks.filter((task) => !task.complete).length} Aufgaben übrig
+      </div>
+      <div className="title_tasklist">
+        <h2>To-Do-List</h2>
+      </div>
+      <div className="back">
+        <div className="value_task">
+          {tasks.map((task) => {
+            return <Task key={task.id} toggleTask={toggleTask} task={task} />;
+          })}
```

Comment:
I usually prefer to write the return of the `map` without the explicit `return` within a `return` statement. Just personal preference though.
```suggestion
          {tasks.map((task) =>  (
           <Task key={task.id} toggleTask={toggleTask} task={task} />;
          ))}
```